docs/further.md

```diff
@@ -282,7 +291,21 @@ snakemake --retries=3
 If a workflow fails entirely (e.g. when there are cluster failures), it can be resumed as any other Snakemake workflow:
 
 ```console
-snakemake --rerun-incomplete
+snakemake ... --rerun-incomplete
+# or the short-hand version
+snakemake ... --ri
+```
+
+The "requeue" option allows jobs to be resubmitted automatically if they fail or are preempted. This might be the default on your cluster, already. You can check your cluster's requeue settings with 
+
+```console
+scontrol show config | grep Requeue
+```
+
+This requeue feature is integrated into the SLURM submission command, adding the --requeue parameter to allow requeuing after node failure or preemption using:
+
+```console
+snakemake --slurm-requeue ...
```

Comment:
How is this different from snakemake's own `--retries` option? Or does this simply implement the `--retries` value for pre-empted jobs or does it simply move the retries to the level of the slurm scheduler? Maybe you can clarify this further?

Reply:
Right. This requires some SLURM background. I tried to explain in a single sentence in commit cb153d2e21f65def715c26f728e9cd98f9be4188 .